src/js/apps/patients/patient/patient_app.js

```diff
@@ -73,6 +75,15 @@ export default SubRouterApp.extend({
       },
     });
 
+    this.listenTo(userActivityCh, 'close:actionSidebar', () => {
+      if (this.getCurrent()._name === 'archive') {
```

Comment:
This app is shared by both the patient dashboard and archive. This is the only way I could think of to differentiate when the app is being used for the dashboard or archive.